### PR TITLE
Support for Wagtail 2.x and remove a bare except.

### PR DESCRIPTION
**Pull request by FlipperPA**

### CODE REVIEW

**Comment by FlipperPA** [comment]
I'm going to go ahead and merge this, as the changes are very minor.

File: wagtailerrorpages/templatetags/wagtailerrorpages_tags.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
 from django import template
+from django.urls import NoReverseMatch
```
Comment (alexgleason) [inline wagtailerrorpages/templatetags/wagtailerrorpages_tags.py]: I think we need a try/except around this too since Django 1.11 uses `django.conf.urls`

Comment (FlipperPA) [reply wagtailerrorpages/templatetags/wagtailerrorpages_tags.py]: GAH, you're absolutely right. That's what I get for merging without review, sorry about that! I'll fix and open a new P.R. and wait this time. :)

**Comment by FlipperPA** [comment]
It actually works in Django 1.11:

```python
In [2]: import django

In [3]: django.VERSION
Out[3]: (1, 11, 10, 'final', 0)

In [4]: from django.urls import NoReverseMatch, reverse

In [5]:
```

However, the other import should be made more granular (separate Python and Wagtail). I'll get you a P.R. in a few minutes. :)

**Pull request merged**